staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -622,3 +531,163 @@ func TestInvalidKeys(t *testing.T) {
 	_, countErr := store.Count(invalidKey)
 	expectInvalidKey("Count", countErr)
 }
+
+func BenchmarkStore_GetList(b *testing.B) {
+	generatePod := func() runtime.Object {
```

Comment:
This object is still fairly small in data.

I would like to see something like "generateBigPod" or sth (and you can approximate it by putting large annotations on it).

Reply:
fixed